Description:
 The package name of okhttp used by Android is "com.android.okhttp", we need to
 use the upstream name instead.
Author: Kai-Chung Yan <seamlikok@gmail.com>
Last-Update: 2016-05-25
--- a/core/java/android/net/Network.java
+++ b/core/java/android/net/Network.java
@@ -36,12 +36,12 @@
 import java.net.URLConnection;
 import javax.net.SocketFactory;
 
-import com.android.okhttp.ConnectionPool;
-import com.android.okhttp.HttpHandler;
-import com.android.okhttp.HttpsHandler;
-import com.android.okhttp.OkHttpClient;
-import com.android.okhttp.OkUrlFactory;
-import com.android.okhttp.internal.Internal;
+import com.squareup.okhttp.ConnectionPool;
+import com.squareup.okhttp.HttpHandler;
+import com.squareup.okhttp.HttpsHandler;
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.OkUrlFactory;
+import com.squareup.okhttp.internal.Internal;
 
 /**
  * Identifies a {@code Network}.  This is supplied to applications via
@@ -65,7 +65,7 @@
     // mLock should be used to control write access to mConnectionPool and mNetwork.
     // maybeInitHttpClient() must be called prior to reading either variable.
     private volatile ConnectionPool mConnectionPool = null;
-    private volatile com.android.okhttp.internal.Network mNetwork = null;
+    private volatile com.squareup.okhttp.internal.Network mNetwork = null;
     private final Object mLock = new Object();
 
     // Default connection pool values. These are evaluated at startup, just
@@ -220,7 +220,7 @@
     private void maybeInitHttpClient() {
         synchronized (mLock) {
             if (mNetwork == null) {
-                mNetwork = new com.android.okhttp.internal.Network() {
+                mNetwork = new com.squareup.okhttp.internal.Network() {
                     @Override
                     public InetAddress[] resolveInetAddresses(String host) throws UnknownHostException {
                         return Network.this.getAllByName(host);
--- a/core/java/android/net/http/HttpResponseCache.java
+++ b/core/java/android/net/http/HttpResponseCache.java
@@ -16,9 +16,9 @@
 
 package android.net.http;
 
-import com.android.okhttp.Cache;
-import com.android.okhttp.AndroidShimResponseCache;
-import com.android.okhttp.OkCacheContainer;
+import com.squareup.okhttp.Cache;
+import com.squareup.okhttp.AndroidShimResponseCache;
+import com.squareup.okhttp.OkCacheContainer;
 
 import java.io.Closeable;
 import java.io.File;
